types/node/test.d.ts

```diff
@@ -1223,6 +1223,10 @@ interface TestFail {
          * The error thrown by the test.
          */
         error: Error;
+        /**
+         * The type of the test, used to denote whether this is a suite
```

Comment:
@philnash is it worth adding the `@since` tag?

Reply:
Hmm. Probably. But, how do you signify multiple versions with `@since`? e.g. this was present in v20.0.0 and then backported to 18.17.0. I've looked around and it seems you can only provide one version to `@since`.

Reply:
I see a few examples with multi versions:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/77c021e1dd7f54c5ad5958bc468fd61a35d1dee7/types/node/crypto.d.ts#L1842